component/sidetree/doc/doc.go

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
sidetree should not be at the root component folder.

Reply:
@troyronda what is the better location for generic sidetree client

Reply:
@troyronda Done. I move it under vdr package